{
  "schema_version": "1.4.0",
  "id": "GHSA-f594-f3v3-g649",
  "modified": "2022-05-17T05:03:06Z",
  "published": "2022-05-17T05:03:06Z",
  "aliases": [
    "CVE-2013-1630"
  ],
  "details": "pyshop before 0.7.1 uses HTTP to retrieve packages from the PyPI repository, and does not perform integrity checks on package contents, which allows man-in-the-middle attackers to execute arbitrary code via a crafted response to a download operation.",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2013-1630"
    },
    {
      "type": "WEB",
      "url": "https://github.com/mardiros/pyshop/commit/ffadb0bcdef1e385884571670210cfd6ba351784"
    },
    {
      "type": "WEB",
      "url": "https://github.com/mardiros/pyshop/blob/master/CHANGES.txt"
    },
    {
      "type": "WEB",
      "url": "http://www.reddit.com/r/Python/comments/17rfh7/warning_dont_use_pip_in_an_untrusted_network_a/"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-20"
    ],
    "severity": "MODERATE",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2013-08-06T02:52:00Z"
  }
}